File: Java/Processor.java
public class Processor
{
    private String name;
    private int price;

    //constructor
    public Processor(){}
    public Processor(String x, int y)
    {
        setName(x);
        setPrice(y);
    }

    //name's methods
    public void setName(String x)
    {
        this.name = x;
    }
    public String getName()
    {
        return this.name;
    }

    //price's methods
    public void setPrice(int y)
    {
        this.price = y;
    }
    public int getPrice()
    {
        return this.price;
    }

}

File: CPP/Processor.cpp
#include <iostream>
#include <string>

using namespace std;

class Processor
{
    private:
        string name;
        int price;

    public:
        
        //constructors
        Processor(){}
        Processor(string x, int y)
        {
            setName(x);
            setPrice(y);
        }
        
        //Name's methods
        void setName(string x)
        {
            this->name = x;
        }
        string getName()
        {
            return this->name;
        }

        //Price's methods
        void setPrice(int y)
        {
            this->price = y;
        }
        int getPrice()
        {
            return this->price;
        }
        
        //destructor
        ~Processor(){}
};